bin/setup.bat

```diff
@@ -15,6 +15,15 @@ goto finally
 
 :setup_jruby
 REM setup_java()
+set JAVA_FROM_SYMLINK=
+if not defined JAVA_HOME IF EXIST %ProgramData%\Oracle\java\javapath\java.exe (
+    for /f "tokens=2 delims=[]" %%a in ('dir %ProgramData%\Oracle\java\javapath\java.exe') do @set JAVA_HOME=%%a
+    set JAVA_FROM_SYMLINK=1
+)
+if defined JAVA_HOME set JAVA_HOME=%JAVA_HOME:\bin\java.exe=%
```

Comment:
This line cuts off the `\bin\java.exe` from the value retrieved from the symlink.
I've added another commit to make it more clear. There I use a variable `JAVA_EXE` and then build `JAVA_HOME` from it.

PTAL